satellite/repair/repair_test.go

```diff
@@ -368,36 +369,36 @@ func TestCorruptDataRepair_Succeed(t *testing.T) {
 			if nodesToKill[node.ID()] {
 				err = planet.StopPeer(node)
 				require.NoError(t, err)
-				_, err = satellite.Overlay.Service.UpdateUptime(ctx, node.ID(), false)
+				_, err = sat.Overlay.Service.UpdateUptime(ctx, node.ID(), false)
 				require.NoError(t, err)
 			}
 		}
 		require.NotNil(t, corruptedNode)
 
 		corruptPieceData(ctx, t, planet, corruptedNode, corruptedPieceID)
 
-		overlay := planet.Satellites[0].Overlay.Service
-		node, err := overlay.Get(ctx, corruptedNodeID)
+		over := planet.Satellites[0].Overlay.Service
```

Comment:
nit: I feel `overlay` is more clear than `over`

Reply:
Same as below